src/cmdInventoryExport.go

```diff
@@ -0,0 +1,148 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+)
+
+type HostVars map[string]interface{}
+type Hosts map[string]HostVars
+
+type Group struct {
+	Hosts []string               `json:"hosts,omitempty"`
+	Vars  map[string]interface{} `json:"vars,omitempty"`
+}
+
+type Groups map[string]Group
+
+type Meta struct {
+	Hostvars Hosts `json:"hostvars"`
+}
+
+type AnsibleInventory struct {
+	Groups Groups `json:"groups"`
+	Meta   Meta   `json:"_meta"`
+}
+
+func (a AnsibleInventory) MarshalJSON() ([]byte, error) {
+	aux := make(map[string]interface{})
+
+	for key, value := range a.Groups {
+		aux[key] = value
+	}
+
+	aux["_meta"] = a.Meta
+
+	return json.Marshal(aux)
+}
+
+type inventoryAnsibleCmd struct {
+	List bool `long:"list" description:"List Ansible Inventory"`
+}
+
+func (c *inventoryAnsibleCmd) Execute(args []string) error {
+	if earlyProcess(args) {
+		return nil
+	}
+	return c.run()
+}
+
+func (c *inventoryAnsibleCmd) run() error {
+	if !c.List {
+		return nil
+	}
+
+	projectName := os.Getenv("PROJECT_NAME")
+
+	inventoryItems := []int{}
+	inventoryItems = append(inventoryItems, InventoryItemClusters)
+	inventoryItems = append(inventoryItems, InventoryItemClients)
+
+	// TODO: do we need this?
+	//if c.opts.Config.Backend.Type == "aws" {
+	//	inventoryItems = append(inventoryItems, InventoryItemAWSAllRegions)
```

Comment:
only if you want aws backend to go through all available region this is real slow. I would not suggest that for the ansible one. It has been implemented to track down who created what in which region.